src/Workspaces/Core/Portable/GeneratedCodeRecognition/GeneratedCodeRecognitionServiceFactory.cs

```diff
@@ -37,12 +37,10 @@ private static bool IsFileNameForGeneratedCode(string fileName)
                 {
                     fileName = Path.GetFileNameWithoutExtension(fileName);
 
-                    if (fileName.EndsWith("AssemblyInfo", StringComparison.OrdinalIgnoreCase) ||
-                        fileName.EndsWith(".designer", StringComparison.OrdinalIgnoreCase) ||
+                    if (fileName.EndsWith(".designer", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
@heejaechang This change is not needed for suppression, it is an unrelated bug fix. This service indicates whether a source file is a temporary generated file, that is likely to change/regenerate during build or in the designer. AssemblyInfo files are not in this category, it was decided in triage that files generated by project templates shouldn't be considered as temporary generated files.